### PR TITLE
fix(VPCEP): vpcep approval resource support import state

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:
fix vpcep approval resource: resource support import; fix document; fix test
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/vpcep' TESTARGS='-run TestAccVPCEndpointApproval_Basic'
...
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccVPCEndpointApproval_Basic -timeout 360m -parallel 4 
=== RUN   TestAccVPCEndpointApproval_Basic 
=== PAUSE TestAccVPCEndpointApproval_Basic
=== CONT  TestAccVPCEndpointApproval_Basic
--- PASS: TestAccVPCEndpointApproval_Basic (204.73s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     204.788s
```

### CODE REVIEW

**Comment by ShiChangkuo** [comment]
@deer-hang it's very appreciate to add more details in the title of PR.

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go [hunk 1]
```diff
@@ -54,80 +59,20 @@ func testAccVPCEndpointApproval_Basic(rName string) string {
...
-    ignore_changes = [enable_dns]
-  }
-}
-
 resource "huaweicloud_vpcep_approval" "approval" {
   service_id = huaweicloud_vpcep_service.test.id
   endpoints  = [huaweicloud_vpcep_endpoint.test.id]
 }
-`, testAccVPCEndpoint_Precondition(rName), rName)
+`, testAccVPCEndpoint_Basic(rName))
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_approval_test.go]: In `testAccVPCEndpoint_Basic`,  **approval** is false, then it's not necessary to approval the connection.
I think we should create a endpoint service with approval = true

**Comment by deer-hang** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/vpcep' TESTARGS='-run TestAccVPCEndpointApproval_Basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccVPCEndpointApproval_Basic -timeout 360m -parallel 4 
=== RUN   TestAccVPCEndpointApproval_Basic 
=== PAUSE TestAccVPCEndpointApproval_Basic
=== CONT  TestAccVPCEndpointApproval_Basic
--- PASS: TestAccVPCEndpointApproval_Basic (205.67s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     205.719s
```

**Pull request merged**